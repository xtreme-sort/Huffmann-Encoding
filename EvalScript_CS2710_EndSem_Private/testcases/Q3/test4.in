GenerateTreeEnhanced
5
H 12
d 34
x 34
6 41
r 48
Quit
