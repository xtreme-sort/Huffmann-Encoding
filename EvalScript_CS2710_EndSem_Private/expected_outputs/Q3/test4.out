((x,6),((H,d),r));
